docs/en/index.md

```diff
@@ -9,14 +9,12 @@ lang: en
 
 Welcome to the documentation center for User Management APIs from Adobe.
 
-**This is a draft development area.  Production documentation is [here](https://www.adobe.io/apis/cloudplatform/usermanagement/docs/gettingstarted.html)**
+**This is a draft development area.**  Production documentation is [here](https://www.adobe.io/apis/cloudplatform/usermanagement/docs/gettingstarted.html).
 
-[Getting Started with User Management](getstarted.md)
+[Getting Started with User Management](getstarted.html)
```

Comment:
Still need to resolve these (and all .html references)

Reply:
This will be taken up in another PR.